core/cli/src/lib.rs

```diff
@@ -52,9 +52,9 @@ pub struct AxonCli {
 }
 
 impl AxonCli {
-    pub fn init(axon_version: Version, cli_version: &'static str) -> Self {
+    pub fn init(axon_version: Version) -> Self {
```

Comment:
The two versions may not be the same for user defined clis using axon as a library.

Reply:
For this question and the below one, I am considering to divide the version into a kernel version and an application version. The current `axon_version` will become the kernel version. If use axon as a binary rather library, the application version will be same as application version.